style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  height: 1000px;
+  justify-content: center;
+}
+
+section {
+  position: absolute;
+  height: 718px;
+  width: 375px;
+  border: 1px solid;
+}
+
+#logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  width: 76px;
+  height: 20px;
+  color: #6070ff;
+  background: rgba(255, 255, 255, 1);
+}
+
+#header-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 130%;
+  justify-content: baseline;
+  align-items: center;
+  color: #172b4d;
+  order: 0;
+  width: 327px;
+  height: 150px;
+  top: 114px;
+  margin-left: 24px;
+  margin-right: 300px;
+  margin-bottom: 12px;
+  margin-top: 110px;
+}
+
+#headline-paragraph {
+  width: 327px;
+  height: 146px;
+  top: 276px;
+  left: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  order: 1;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+#contact-link {
+  width: 117px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
+  color: #7f8cff;
+  order: 0;
+  margin-left: 24px;
+  text-decoration: none;
+}
+
+#vector-1 {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  border: 2px #acb7c3;
+  margin-left: 270px;
+  margin-top: -40px;
+}
+
+.contact-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style-type: none;
+  margin-left: 24px;
+  margin-right: 180.83px;
+  width: 200px;
+}
```

Comment:
- [ ] As suggested by the previous reviewer please adjust the alignment of the social media icons to make them look identical to the figma design template

Figma design:

<img width="137" alt="Screenshot 2023-03-25 at 9 19 01 AM" src="https://user-images.githubusercontent.com/44798044/227696667-05cb8116-b5ae-408f-9ca9-5ddbef531080.png">


Your design:

<img width="338" alt="Screenshot 2023-03-25 at 8 40 26 AM" src="https://user-images.githubusercontent.com/44798044/227696718-834ab4b5-cb32-43bb-a429-62c3c3b3cf6d.png">